Mathlib/MeasureTheory/Measure/MeasureSpace.lean

```diff
@@ -1228,8 +1228,10 @@ theorem tendsto_ae_map {f : α → β} (hf : AEMeasurable f μ) : Tendsto f (ae
 /-- Pullback of a `Measure` as a linear map. If `f` sends each measurable set to a measurable
 set, then for each measurable set `s` we have `comapₗ f μ s = μ (f '' s)`.
 
+Note that if `f` is not injective, this definition assigns `Set.univ` measure zero.
```

Comment:
I like these changes, but they modify a completely different file. Can you remove them from this PR? Feel free to submit a new PR with them; if you @mention me, I will do my best to review it quickly.

Reply:
Oops, these changes were part of an earlier PR that was approved. Didn't mean to include them again! Maybe what happened was that I wrote
`git switch -c bjoernkjoshanssen_measure_comap_injectivity`
back in July and I wrote
`git switch -c bjoernkjoshanssen_onepoint`
yesterday, but I should have done
`git switch -c master`
in between?

Reply:
That sounds plausible; running `git switch -c branchname` creates a new branch based on the branch you currently have checked out. Switching to `master` before running this ensures your branch is based on master (which is what you want here).
(Sometimes, the other behaviour, branching your branch on top of an existing branch, is precisely what you want. It depends.)